Cargo.toml

```diff
@@ -1,8 +1,10 @@
 [package]
 name = "mongodb"
 version = "0.1.0"
-authors = ["Valeri Karpov <valkar207@gmail.com>"]
+authors = ["Valeri Karpov <valkar207@gmail.com>",
+           "Saghm Rossi <saghmrossi@gmail.com",
+           "Kevin Yeh <kevinyeah@utexas.edu"]
 
 [dependencies]
-
-libc="0.1.8"
+byteorder = "*"
```

Comment:
Ah, okay, sorry about that. I was just copying the dependency line that they suggested on github.